dev/org.eclipse.codewind.openapi.core/src/org/eclipse/codewind/openapi/core/IOpenApiConstants.java

```diff
@@ -23,4 +23,7 @@
 	public static final String CODEWIND_PROJECT_LANGUAGE = "projectLanguage"; //$NON-NLS-1$
 	public static final String CODEWIND_GET_ID_METHOD = "getId"; //$NON-NLS-1$
 	public static final String CODEWIND_CW_SETTINGS = ".cw-settings"; //$NON-NLS-1$
+	
+	public static String CONTENTTYPE_YAML = "org.eclipse.codewind.openapi.core.openApiDocumentYaml";
```

Comment:
Do we need to add the `//$NON-NLS-1$` anymore? Or are we stopping that?

Reply:
@sghung, added.